experimental/dds/sequence-deprecated/src/sparsematrix.ts

```diff
@@ -90,7 +89,7 @@ export class PaddingSegment extends BaseSegment {
  * @deprecated `SparseMatrixItem` is part of an abandoned prototype.
  * Use {@link @fluidframework/matrix#SharedMatrix} instead.
  */
-export type SparseMatrixItem = Serializable;
+export type SparseMatrixItem = any;
```

Comment:
I understand that before your change, Jsonable/Serializable without a generic argument evaluated to the 'any' type.

I thought your change made Jsonable/Serializable<unknown> produce a useful constraint, and it does seem like you use it that that way below...

...So I'm confused as to why you're not using it here?

Reply:
Because this is deprecated and simply chose to keep the type the same.